docs_website/docs/configurations/infra_config.md

```diff
@@ -98,6 +98,8 @@ the next few configurations are only relevant if you are using OAuth based authe
 -   `OAUTH_AUTHORIZATION_URL` (**required**): Url for oauth redirection
 -   `OAUTH_TOKEN_URL` (**required**): Url to get the oauth token
 -   `OAUTH_USER_PROFILE` (**required**): Url to get the user profile
+-   `OAUTH_TOKEN_PLACE`: Place to send the obtained access_token (headers) or default if not informed (url params)
+-   `OAUTH_TOKEN_HEADER_PREFIX`: Prefix to add to the Authorization header (i.e: Bearer)
```

Comment:
Hm do you know if there is any case where the format is not "authorization: bearer <token>"?This option might not be necessary?

Reply:
Removed in the last commit. By default, it would be in the headers.